docs/en/patterns/contextual-menu.md

```diff
@@ -0,0 +1,13 @@
+---
+collection: patterns
+title: Contextual menu
+---
+
+## Contextual menu
+
+A contextual menu can be applied to any button, link or navigation item that requires a secondary menu.
```

Comment:
Please add JavaScript documentation. With details on the hooks and how it works.